adafruit_dymoscale.py

```diff
@@ -43,7 +44,12 @@ class ScaleReading:
 class DYMOScale:
     """Interface to a DYMO postal scale."""
 
-    def __init__(self, data_pin, units_pin, timeout=1.0):
+    def __init__(
+        self,
+        data_pin: microcontroller.pin,
```

Comment:
I'm a little surprised this didn't raise a CI error!  `data_pin` should be type annotated as `microcontroller.Pin` :)